CentralUnit/Models/Lap.cs

```diff
@@ -51,6 +36,31 @@ public void AddPenalties(List<PenaltyEvent> penalties)
             this.penalties.AddRange(penalties);
         }
 
+        /// <summary>
+        /// Get lap time in microseconds. Laps with a DNF get a lap time of -1
+        /// Laps with a DSQ get a normally calulcated lap time
+        /// </summary>
+        /// <param name="includePenalties">if true add the lap penalties to the calculated lap time</param>
+        /// <returns></returns>
+        public long GetLapTime(bool includePenalties=true)
+        {
+            if (End is FinishedEvent f)
+            {
+                if (includePenalties)
+                {
+                    return f.LapTime + (penalties.Sum(p => p.Seconds) * 1000000);//1 penalty second is 1000000 microseconds
```

Comment:
Isn't it easier to have a MicroSeconds property in the Penalty Event?

Reply:
I'd say we store times in the resolution we receive them. We get microseconds from the timing system, but people enter penalties in seconds.

If this gets out of hand we can decide to store all times in the same units (micro) and keep all the conversions at the i/o methods

Reply:
The point was to store it in seconds and have MicroSeconds as a read-only property.